.github/workflows/main.yml

```diff
@@ -211,6 +244,7 @@ jobs:
   coveralls:
     name: Indicate completion to coveralls.io
     needs: [conda-build, pip-build, hindcast-calls]
+    if: always() && (needs.hindcast-calls.result == 'success' || needs.hindcast-calls.result == 'skipped')
```

Comment:
Is this indicating that if the hindcast tests are run but fail, coveralls reporting is skipped? Will coveralls still consider the hindcast module to be covered if the hindcast tests don't need to run?

Reply:
If hindcast fails coveralls is skipped. You can see that as the current behavior:
![image](https://github.com/MHKiT-Software/MHKiT-Python/assets/13438942/a3aabd56-6492-44b5-a169-36d8107f8d68)


In my screenshot for this PR description I show the used logic. You can see that hindcast is skipped and that coveralls is submitted.

Reply:
What coveralls does is consider lines of code ran vs lines of code tested and aggregating that into a single number. 

When we do not run hindcast the lines do not count against coverage (in our case we will catch this in commits to master). 

Coverage is always a tenuous metric at best.